src/main/app/src/app/shared/edit/edit-select/edit-select.component.ts

```diff
@@ -28,7 +28,7 @@ export class EditVertrouwelijkheidaanduidingComponent extends EditComponent {
 
     init(formField: SelectFormField): void {
         this.option = formField.formControl.value;
-        this.value = formField.formControl.value.value;
+        this.value = formField.formControl.value.value ? formField.formControl.value.value : formField.formControl.value[formField.optionLabel];
```

Comment:
Moet zoiets zijn denk ik:

this.value = formField.formControl.value ? formField.formControl.value[formField.optionLabel] : formField.formControl.value;